bob/mkosi.extra/etc/apparmor.d/searcher-container

```diff
@@ -0,0 +1,46 @@
+#include <tunables/global>
+
+profile searcher-container flags=(attach_disconnected,mediate_deleted) {
+  #include <abstractions/base>
+
+  network,
+  capability,
+  file,
+  umount,
+
+  # Allow signals from privileged profiles and from within the same profile
+  signal (receive) peer=unconfined,
+  signal (send,receive) peer=searcher-container,
+  # Allow signals from podman/runc/crun
+  signal (receive) peer={/usr/bin/,/usr/sbin/,}{podman,runc,crun*},
+  
+  deny @{PROC}/* w,   # deny write for all files directly in /proc (not in a subdir)
+  # deny write to files not in /proc/<number>/** or /proc/sys/**
+  deny @{PROC}/{[^1-9],[^1-9][^0-9],[^1-9s][^0-9y][^0-9s],[^1-9][^0-9][^0-9][^0-9/]*}/** w,
```

Comment:
PID might have more than 4 digits

Reply:
It supports PIDs w/ > 4 digits using the `[^0-9/]*`. But this regex is used across the board with Podman, Docker, etc in all of their apparmor profile generators, so I think it's pretty standard.